helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -1056,6 +1056,15 @@ public Job removeJob(final JobId id, final String token)
       throw new HeliosRuntimeException("removing job " + id + " failed", e);
     }
 
+    // Delete job history on a best effort basis
+    try {
+      client.deleteRecursive(Paths.historyJob(id));
```

Comment:
just curious - is it not possible to have the recursive delete added to the transaction above? or just not desired in this case?